client/src/components/CreateCardForm/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Card, CardBody, Input } from 'mdbreact'
+
+const CreateCardForm = props => (
+  <div className="col-sm-3 mx-auto">
+    <Card>
+      <CardBody>
+        <form className="text-left" onSubmit={props.handleSubmit}>
+          <Input
+            value={props.description}
+            onChange={props.handleChange}
+            icon="list prefix grey-text left"
+            label="What to do?"
+          />
+          <div className="col-sm-4 mx-auto">
+            <Input
+              type='submit'
+              value='Add'
```

Comment:
@datarockets/cuties thoughts about using `'` or `"` everywhere?

Reply:
@AleksSenkou I prefer "" because lots of boilerplate code and tutoruals use doublequotes

Reply:
@nikitasakov what about you?

Reply:
from my side: I'm ok with both

Reply:
We can discuss it in parallel and create card for updating linter in order not to block the PR

Reply:
https://trello.com/c/uZc2dbSs/24-when-i-develop-the-app-id-like-strings-to-be-consistent-across-the-app-so-id-like-to-use-single-double-quarters-everywhere

Leave your thoughts there